Swish.xcodeproj/project.pbxproj

```diff
@@ -80,9 +81,8 @@
 		F806950F1B962C5300C61B4A /* RequestPerformer.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = RequestPerformer.swift; sourceTree = "<group>"; };
 		F80695111B962C5300C61B4A /* Info.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		F80695121B962C5300C61B4A /* Swish.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = Swish.h; sourceTree = "<group>"; };
-		F80695171B9633C400C61B4A /* Result.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Result.framework; path = "Carthage/Checkouts/Result/build/Debug-iphoneos/Result.framework"; sourceTree = "<group>"; };
```

Comment:
All I did was check this box.

<img width="260" alt="screen shot 2015-10-20 at 6 52 33 pm" src="https://cloud.githubusercontent.com/assets/84700/10625749/cb55eabc-775b-11e5-918c-ae174e1d0826.png">

I guess this should have been removed as part of some previous changes but wasn't. Classic Xcode.